libs/template/docs/api/let-directive.md

```diff
@@ -103,6 +103,47 @@ Internally, `*rxLet` is using a simple "view memoization" - it caches all anchor
 them whenever the observable notification (next/error/complete) is sent. Then, it only updates the context
 (e.g. a value from the observable) in the view.
 
+## Testing
+
+### Handling the scheduling issue
+
+By default `*rxLet` uses the normal concurrent strategy which renders the view using a scheduling mechanism. This behavior leads to unexpected results in a test environnement, that's why we recommend to test your templates using the native strategy. This is configurable with the following provider.
+
+```ts
+export const RX_ANGULAR_TEST_PROVIDER: StaticProvider = {
+  provide: RX_RENDER_STRATEGIES_CONFIG,
+  useValue: {
+    primaryStrategy: 'native',
+  },
+};
+```
+
+Then, declare it in the providers entry.
+
+```ts
+TestBed.configureTestingModule({
+  ...
+  providers: [RX_ANGULAR_TEST_PROVIDER],
+}).compileComponents();
+```
+
+This way, `*rxLet` will use the same rendering strategy used by the Angular's built-in `async` pipe.
+
+### Handling the lazy rendering problem
+
+Keep in mind that no matter which strategy is used, `*rxLet` always renders the view asynchronously. For this reason, you want to be sure the asynchronous change detection is done before querying any DOM element rendered by the directive. You can achieve this using the `ComponentFixture#whenStable()` function.
```

Comment:
this should not be true. The provided strategy itself is accountable for the let directive rendering sync or async. e.g. `native` has no built in scheduling or other asynchronous calls, thus it will instantly call `cdRef#markForCheck`.

maybe for testing we should even provide a very new strategy which is sync and calls `detectChanges` instead. This would probably result in the desired behavior